modules/nux/modules.php

```diff
@@ -68,11 +68,13 @@ public function process() {
         $smtp_servers = NULL;
         $feed_servers = NULL;
         $profiles = NULL;
+        $jmap_servers = NULL;
 
         $modules = $this->config->get_modules();
 
         if (data_source_available($modules, 'imap')) {
-            $imap_servers = count(Hm_IMAP_List::dump(false));
+            $imap_servers = count(array_filter(Hm_IMAP_List::dump(false), function($v) { return !array_key_exists('type', $v);}));
```

Comment:
To get imap you can just do:
'Hm_IMAP_List::dump(false) - $jmap_servers' instead of filtering again